core/core-awt/src/main/java/org/icepdf/core/pobjects/CrossReference.java

```diff
@@ -77,7 +77,7 @@ public void setTrailer(PTrailer trailer) {
     public int getNextAvailableReferenceNumber() {
         List<Integer> objectNumbers = Collections.list(hObjectNumber2Entry.keys());
         Collections.sort(objectNumbers);
-        return objectNumbers.get(objectNumbers.size() - 1) + 1;
+        return objectNumbers.isEmpty() ? 0 : objectNumbers.get(objectNumbers.size() - 1) + 1;
```

Comment:
I think in this particular case an empty list should return 1, object numbers have to be a positive integer.    
I didnt' think of this corner case, I'm guessing you've been seeing it a lot?

Reply:
Okay, I set it to 1.      
Not really, it has only been one pdf file so far, but we don't have a lot of active users at the moment.

Reply:
definitely hit this a few times during the font work.  Like your code better so going with it.